sdk/communication/azure-communication-chat/env-prod.properties

```diff
@@ -11,7 +11,7 @@ trouter.registration.hostname="teams.microsoft.com/registrar/prod"
 trouter.registration.hostname.dod="registrar.dod.teams.microsoft.us"
 trouter.registration.hostname.gcch="registrar.gov.teams.microsoft.us"
 trouter.application.id="AcsAndroid"
-trouter.template.key="AcsAndroid_Chat_1.3"
+trouter.template.key="AcsAndroid_Chat_1.7"
```

Comment:
one question, it seems like Ali has added attachments to 1.7 which we don't plan to support right now, can you find a template version that doesn't have this attachments please?

Reply:
maybe 1.5? would you mind testing it and see if everything is working fine with 1.5 please?

Reply:
Changed to 1.5 and tested locally